hax/hax/util.py

```diff
@@ -62,13 +62,14 @@
 MotrConsulProcInfo = NamedTuple('MotrConsulProcInfo', [('proc_status', str),
                                                        ('proc_type', str)])
 
-MotrConsulProcStatus = NamedTuple('MotrConsulProcStatus', [(
-                                        'consul_svc_status', str),
-                                        ('consul_motr_proc_status', str)])
+MotrConsulProcStatus = NamedTuple('MotrConsulProcStatus',
```

Comment:
Through out Hare code we follow https://peps.python.org/pep-0008/#indentation guide lines. So opening brackets must be on the same line.

Reply:
@mssawant  Yes following pep8 style. To keep the indentation like the previous statement, the list argument's [ bracket was brought down. If I pull "[" up then the rest of array elements will be moved like this.
faultToleranceInfo = NamedTuple('faultToleranceInfo', [
    ('max_drives_per_node', int),
    ('max_drives_per_ctrl', int),
    ('min_drives_per_node', int)])

Reply:
Fixed.